docs/storage.md

```diff
@@ -425,11 +426,19 @@ config:
   http_config:
     idle_conn_timeout: 1m30s
     response_header_timeout: 2m
+    insecure_skip_verify: false
     tls_handshake_timeout: 10s
     expect_continue_timeout: 1s
     max_idle_conns: 100
     max_idle_conns_per_host: 100
     max_conns_per_host: 0
+    tls_config:
```

Comment:
I hope COS can laverage our TLS config 🤔 If we care our transport, I bet it can (:

Reply:
The tlsconfig is now common for all the three ie.cos azure and s3 because we have the same struct. It was not there for cos but now it is, so do I need to find a way to remove it, am I missing something?

Reply:
Yes, my only point is, does COS  actually use that TLSConfig field from our config?

Reply:
I checked and all good: https://github.com/thanos-io/thanos/pull/5447/files#diff-b16f6b19621647e8d48e2142fd1bca0ed1a0a6b154a342d4c8198648edd32a5fR133